Essentials/src/com/earth2me/essentials/commands/Commanddelkit.java

```diff
@@ -0,0 +1,29 @@
+package com.earth2me.essentials.commands;
+
+import com.earth2me.essentials.User;
+
+import org.bukkit.Server;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class Commanddelkit extends EssentialsCommand {
+
+    public Commanddelkit() {
+        super("deletekit");
+    }
+
+    @Override
+    public void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
+        if (args.length != 1) {
+            throw new NotEnoughArgumentsException();
+        }
+
+        String kitname = args[0];
+        if (!ess.getSettings().isPastebinCreateKit()) {
+            ess.getKits().removeKit(kitname);
+            user.sendMessage(tl("deletedKit", kitname));
+        } else {
+            user.sendMessage("You have tried to remove the kit from pastebin.com which is not very easy.");
```

Comment:
1. If the user wants to delete the kit, they want to delete the kit. I don't see why they shouldn't be able to delete the kit just because they didn't add it via pastebin.

2. Always use translatable messages.

Reply:
Actually, I don't know what API for the pastebin is, so we can delete the added kit list.